openwisp_radius/counters/base.py

```diff
@@ -21,9 +21,17 @@ def check_name(self):  # pragma: no cover
         pass
 
     @property
-    @abstractmethod
-    def reply_name(self):  # pragma: no cover
-        pass
+    def reply_names(self):
```

Comment:
I don't understand very well if this method is really needed here? Where is the logic below used?